packages/components/src/components/tabs/tabs.scss

```
@@ -85,9 +88,28 @@
             width: var(--tab-width);
         }
     }
+
     &__active {
         font-weight: 700;
         color: var(--text-prominent);
+
+        &:after {
+            border-bottom: 2px solid var(--brand-red-coral);
+            content: '';
+            position: absolute;
+            bottom: -5px;
+            left: 0;
+            display: block;
+            width: 100%;
+
+            @media (min-width: 325px) {
```

You could `mobile-up` mixin for this (unless the extra 5 pixels are important).

```suggestion
            @include mobile-up {
```